backend/lib/models/schemas/Feedback.js

```diff
@@ -0,0 +1,39 @@
+const { Schema } = require("mongoose");
+const { locationSchema } = require("./Location");
```

Comment:
This is requiring the old location schema, which was blank. 
We should probably replace this with the defined location schema.

Reply:
is that schema on master or is that still on a separate branch @Naraujo13 ?